MetaMask/metamask-extension
# Dark Mode: Create Theme-friendly top left logo component

Explanation:  The top left horizontal logo having hardcoded text colors is problematic.  This PR moves the standard, flask, and beta build type logos into one component and changes their font colors accordingly.

Manual testing steps:  
  - Open MetaMask in full screen mode
  - Look at the logo text in both standard and dark modes
  - Manually open `ui/components/ui/metafox-logo/horizontal-logo.js` and change `process.env.METAMASK_BUILD_TYPE` to `flask` and `beta`, ensure they look good in standard and dark mode
